docs/static/releasenotes.asciidoc

```diff
@@ -68,6 +69,33 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-15-4]]
+=== Logstash 8.15.4 Release Notes
+
+[[notable-8.15.4]]
+==== Notable issues fixed
+
+* Fixed an issue where Logstash could not consume lines correctly when the input buffer was full while utilizing a codec with a delimiter https://github.com/elastic/logstash/pull/16482[#16482]
```

Comment:
@andsel Does this line properly describe the bugfix of BufferedTokenizer?

Reply:
To me seem great 👍

Reply:
Can we consider this: 

"**Fixed an issue where Logstash could not consume lines correctly when a codec with a delimiter is in use and the input buffer becomes full.**"

This suggestion puts the more permanent condition first (codec with a delimiter) and the more transitional condition (buffer fills up) later.

Reply:
It works. Applied the suggestion